config/schema/oe_contact_forms.schema.yml

```diff
@@ -0,0 +1,55 @@
+# Schema for the configuration files of the OE Contact Forms module.
+
+contact.form.*.third_party.oe_contact_forms:
+  type: mapping
+  label: 'OE Contact Forms settings'
+  mapping:
+    is_corporate_form:
+      type: boolean
+      label: 'Is corporate form'
+    topic_label:
+      type: label
+      label: 'Topic label'
+      translatable: true
```

Comment:
The `label` type is already translatable so you can remove from here. Same for all the others below please (and text)

Reply:
Fixed after checking `core/config/schema/core.data_types.schema.yml`